lisa/tools/hyperv.py

```diff
@@ -445,6 +445,71 @@ def enable_internal_dhcp(self, dhcp_scope_name: str = "DHCPInternalNAT") -> None
         # Restart the DHCP server to apply the changes
         service.restart_service("dhcpserver")
 
+    # The default lisa working path is under 'C:/' which is not ideal for large data.
```

Comment:
Use a configurable path, instead of use the LISA working path. The big files should be tracked separately.

Reply:
You can use LISA working path as the default path, but it should be configurable for lab environments.